panopticon/datadog.py

```diff
@@ -57,6 +99,8 @@ def stats(cls):
         This will return a `mock.Mock` instance if the `DATADOG_ENABLED` setting
         is `False`. This makes it possible to run this in development without
         having to make any additional changes or conditional checks.
+
+        :return datadog.ThreadState
```

Comment:
I'm generally an advocate of the [Google Docstring Style](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) because it's more readable in the actual docstring and tools like sphinx understand both. Does that apply to PyCharm as well? Or does it have to be this notation?